generated/niscope/session.py

```diff
@@ -777,17 +745,14 @@ class _SessionBase(object):
     '''Type: enums.FIRFilterWindow
 
     Specifies the FIR window type. The possible choices are:
-    NISCOPE_VAL_NONE
+    _FIRFilterWindow.NONE
```

Comment:
It looks like the documentation is incorrect for the other values.

This attribute is private though which is probably why we never noticed.

For some reason the NONE value was not being fixed up before, which is why it changed from `NISCOPE_VAL_NONE` to `_FIRFilterWindow.NONE`. 

The reason for the '_' prefix is the the attribute is private and since that is the only use of the enum, it is also private.